app/state/app/app.reducer.js

```diff
@@ -19,6 +20,11 @@ export default (state = initialState, action = {}) => {
         ...state,
         apiHost: initialState.apiHost,
       };
+    case APP_CONSTANTS.SET_SKIN:
+      return {
+        ...state,
+        skin: action.payload,
+      };
```

Comment:
I suggest writing a simple unit test to test that the `setSkin` action creator updates the `app` reducer appropriately. There should be some examples in `app.reducer.test.js` and you can run all the unit the tests with `yarn test`